src/chrome/logging/methodsCalledLogger.ts

```diff
@@ -128,17 +132,17 @@ export class MethodsCalledLogger<T extends object> {
     }
 
     private logCallStart(propertyKey: PropertyKey, methodCallArguments: any[], callId: number): void {
-        const message = `START            ${callId}: ${this.printMethodCall(propertyKey, methodCallArguments)}`;
+        const message = `START ${callId}: ${this.printMethodCall(propertyKey, methodCallArguments)}`;
         logger.verbose(message);
     }
 
-    private logSyncPartFinished(propertyKey: PropertyKey, methodCallArguments: any[], callId: number): void {
-        const message = `PROMISE-RETURNED ${callId}: ${this.printMethodCall(propertyKey, methodCallArguments)}`;
+    private logSyncPartFinished(propertyKey: PropertyKey, _methodCallArguments: any[], callId: number): void {
+        const message = `SNRET ${callId}: ${this.printMethodName(propertyKey)}`;
```

Comment:
what does SNRET mean?

Reply:
It's my abbreviation of synchronic return